src/azure/cli/commands/parameters.py

```diff
@@ -65,6 +65,9 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
         return list(enum_type.__dict__[ENUM_MAPPING_KEY].keys())
     return completer
 
+def get_enum_choices(enum_type):
+    return [x.value for x in enum_type]
```

Comment:
Can you see if the completer can be simplified to simply return the result of "get_enum_choices"? It looks like it can. 

---
In reply to: [74156155](https://github.com/Azure/azure-cli/pull/668#discussion_r74156155) [](ancestors = 74156155,74151895)